examples/layers/.gitignore

```diff
@@ -0,0 +1,10 @@
+.DS_Store
```

Comment:
cc @mit-mit for review of the changes to the examples/layers/ directory to support gradle builds